src/Modules/UngradedModules.jl

```diff
@@ -1632,8 +1636,13 @@ parent_type(::SubQuoElem{T}) where {T} = SubQuo{T}
 elem_type(::Type{SubQuo{T}}) where {T} = SubQuoElem{T}
 parent_type(::Type{SubQuoElem{T}}) where {T} = SubQuo{T}
 
-function in(v::SubQuoElem, M::ModuleFP)
-  return parent(v) === M
+function in(::SubQuoElem, ::FreeMod)
+  return false
```

Comment:
In which situation is this helpful? Shouldn't it rather throw a "type mismatch" error or so?

Anyway, if it is kept: Not covered by tests, could we get a simple one?